src/hooks/core/useCoreChatSystemHook.ts

```diff
@@ -40,6 +46,8 @@ const useCoreChatSystemHook = () => {
   const [migrationStatus] = useRecoilState(migrationDbStatusAtom);
   const initialStartup: any = useRecoilValue(InitialStartupAtom);
   const {params} = useRoute();
+  const [replyPreview, dispatch] = (React.useContext(Context) as unknown as any).chat;
+  const [processedMessageId, setProcessedMessageId] = React.useState<string | null>(null);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `any` type on line 49 is not recommended as it can lead to potential runtime errors. It would be better to define a specific type for the context.



<!-- This is an auto-generated comment by OSS CodeRabbit -->